internal/cloudavenue/provider.go

```diff
@@ -59,6 +62,12 @@ func (p *cloudavenueProvider) Schema(_ context.Context, _ provider.SchemaRequest
 			"url": schema.StringAttribute{
 				MarkdownDescription: "The URL of the CloudAvenue API. Can also be set with the `CLOUDAVENUE_URL` environment variable.",
 				Optional:            true,
+				Validators: []validator.String{
+					stringvalidator.RegexMatches(
+						regexp.MustCompile(`[a-z]$`),
```

Comment:
[This regex](https://regex101.com/r/jltDM2/1) not working for URL format and not checking if string is URL format.

Try this regex ([Regex101](https://regex101.com/r/jG5VyL/1))
`^(https|http):\/\/([\da-z\.-]+)\.([a-z\.]{2,6})(:[\d]{2,5})?([\/\w \.-]*)[a-z0-9]+$`

Reply:
Or just `^https?:\/\/\S+\w$` ?
https://regex101.com/r/QaGBU3/1

Reply:
Corrected